src/Game.java

```diff
@@ -1,13 +1,64 @@
-import java.util.ArrayList;
+import java.util.Date;
+import java.util.Random;
+import java.util.Scanner;
 
 public class Game {
-  static ArrayList<Zombie> zombies = new ArrayList<Zombie>();
   static Board board = new Board(4, 4);
 
+  private static void printOptions() {
+    System.out.println("1: Place sunflower");
+    System.out.println("2: Place peashooter");
+    System.out.println("3: Skip Turn");
+    System.out.println("4: Quit");
+    System.out.print("Enter option number of what you would like to do: ");
+  }
+
+  private static PieceLocation parseCoordinates(String input){
+    //split coordinates by comma and remove any spaces
+    String[] locationSplit = input.split(",");
+    int row = Integer.parseInt(locationSplit[0].trim());
+    int col = Integer.parseInt(locationSplit[1].trim());
+    if(row > 3 || col > 3){
+      throw new IllegalArgumentException("Out of range");
+    }
+    return new PieceLocation(row, col);
+  }
+
   public static void main(String args[]) {
-    Zombie zombie = new Zombie();
-    board.print();
-		board.placePiece(zombie, 1, 1);
-    board.print();
+    Scanner userInput = new Scanner(System.in);
+    Date lastTick = new Date();
+    Random random = new Random();
+    PieceLocation newLocation;
+
+    while(true) {
+      board.print();
+
+      printOptions();
+      String input = userInput.nextLine();
+
+      switch(input) {
+        case "1":
+          System.out.print("Enter where to place sunflower ex(row,col): ");
+          newLocation = parseCoordinates(userInput.nextLine());
+          Sunflower sunflower = new Sunflower();
+          board.placePiece(sunflower, newLocation.getRow(), newLocation.getCol());
+          break;
+        case "2":
+          System.out.print("Enter where to place peashooter ex(row,col): ");
+          newLocation = parseCoordinates(userInput.nextLine());
+          Peashooter peashooter = new Peashooter();
+          board.placePiece(peashooter, newLocation.getRow(), newLocation.getCol());
+          break;
+        case "4":
+          return;
+        default:
+          // If it has been longer than 1 seconds
+          if ((Math.abs(new Date().getTime() - lastTick.getTime())/1000) > 1) {
```

Comment:
need to update lastTick

Reply:
also i dont think this is a good way to do it, since zombies should spawn at certain intervals, and each plant should cost a certain 'sun' point value which is generated at a certain interval. i think for now at the end of each turn, a certain amount of sun points will be added to the player's total sun points and maybe if we dont want to hard code a level we can have it so that we spawn N number of zombies and have a random seed that will generate them with a certain chance per turn until there are 0 zombies left to spawn

Reply:
wtf are sunpoints

Reply:
^currently adding sun point stuff

Reply:
sunpoints = cash to buy flowers

Reply:
@VikramBombhi basically its the currency you use to buy flowers, you get them every once in a while and its what the sunflower plant generates cause they dont attack